mlir/test/CMakeLists.txt

```diff
@@ -31,12 +31,14 @@ if (MLIR_INCLUDE_INTEGRATION_TESTS)
   option(MLIR_RUN_AMX_TESTS "Run AMX tests.")
   option(MLIR_RUN_X86VECTOR_TESTS "Run X86Vector tests.")
   option(MLIR_RUN_CUDA_TENSOR_CORE_TESTS "Run CUDA Tensor core WMMA tests.")
+  option(MLIR_RUN_ROCM_WMMA_TESTS "Run WMMA tests for AMD GPU.")
```

Comment:
I'm not sure this is what we want? Could we instead, since the integration tests already know what chipset we're running on, autodetect whether the WMMA ops are available on the current architecture and make running the WMMA tests a derived property instead of an option people will forget about?